azext_iot/operations/hub.py

```diff
@@ -734,10 +741,10 @@ def iot_hub_configuration_delete(cmd, config_id, hub_name=None, resource_group_n
         raise CLIError(err)
 
 
-def iot_edge_deployment_metric_show(cmd, config_id, metric_id,
+def iot_edge_deployment_metric_show(cmd, config_id, metric_id, metric_type='user',
```

Comment:
Is this a requirement change

Reply:
We talked about this - so we can ensure both the configuration and edge show-metric work exactly the same. We'd have to make a behavior change either way since the old version of edge show-metric was specifically for system metrics.